tests/integration/components/validated-form-test.js

```diff
@@ -132,9 +132,13 @@ test('it renders submit buttons', function(assert) {
 });
 
 test('it renders a hint using a different class from the help block', function(assert){
-  /* this test depends on having 'help' and 'hint' set to different
-     class names in config/environment.js. I can't figure out how to
-     mock that file... */
+  this.set('config', {
+    css: {
+      help: 'help',
+      hint: 'hint'
+    }
+  });
+  
   this.render(hbs`
     {{#validated-form as |f|}}
```

Comment:
Hi @jacob-financeit - thanks for polishing this PR :+1: 
I think this is the only change which is still missing.